docs/api-details/custom-encryption-keys.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Customer Managed Encryption Keys'
+---
+
+<Info>
+To use this offering, you will need:
+- A premium tier Vital subscription
```

Comment:
Enterprise *

Reply:
Depending on thales discussion this will need updating as well